examples/src/examples/graphics/mrt.tsx

```diff
@@ -0,0 +1,201 @@
+import * as pc from '../../../../';
+
+class MrtExample {
+    static CATEGORY = 'Graphics';
+    static NAME = 'Multi Render Targets';
+    static WEBGPU_ENABLED = true;
+
+    static FILES = {
+
+        // shader chunk which outputs to multiple render targets
+        // Note: gl_FragColor is not modified, and so the forward pass output is used for target 0
+        'output.frag': /* glsl */`
+            #ifdef MYMRT_PASS
+                // output world normal to target 1
+                gl_FragColor1 = vec4(litShaderArgs.worldNormal * 0.5 + 0.5, 1.0);
```

Comment:
Should we use the new pc alias for frag color here maybe to illustrate how it should be done?

Reply:
it uses new `gl_FragColor1` here?